tests/test_standard_tags.py

```diff
@@ -289,7 +289,7 @@ def test_client_ip(self):
     def setup_client_ip_vendor(self):
         self._setup_without_attack()
 
-    @bug(library="golang", reason="missing cf-connecting-ipv6")
+    @bug(context.library < "golang@1.69.0", reason="missing cf-connecting-ipv6")
```

Comment:
Is there a jira for this bug? if yes, can you replace `missing cf-connecting-ipv6` by the jira ticket ? 

Otherwise, can you use `APMRP-360` ? (it's our "cold-case" ticket).

Reply:
@cbeauchesne , no jira for this. But the annotation is meant to imply "a fix was added in 1.69.0" which is the upcoming release; like I added support for cf-connecting-ipv6 in a recent PR which will land in 1.69.0.
How do you recommend I proceed, then? Just use APMRP-360?

Reply:
yep : 

```suggestion
    @bug(context.library < "golang@1.69.0", reason="APMRP-360")
```